### ISSUE TITLE
Super weird bug with onPress

### ISSUE THREAD

**Issue by spyshower** [react-native-community/react-native-svg#990]
Ok so I want to present you a really weird bug. This is my code:

```
<View style={styles.photosContainer}>

          <View>
            {this.state.mainUserProfile && this.state.images.length >= 1 ?
              <TouchableOpacity onPress={() => { this.navigateToViewPicture(0) }}>
                
                <Image source={{ uri: 'http://res.cloudinary.com/dy6bbey4u/image/upload/' + this.state.images[0]}}
                  style={styles.picture}/>
              </TouchableOpacity>
              :
              <TouchableOpacity onPress={() => { this.openPickImageModal(0) }} style={styles.placeholderPicture}>
                <ImagePlaceholder style={{ width: pictureWidth, height: pictureWidth }} pictureWidth={pictureWidth} svgPlusNegativeWidth={pictureWidth/5}/>
              </TouchableOpacity>
            }

          </View>

          <View >
            {this.state.mainUserProfile && this.state.images.length >= 2 ?
              <TouchableOpacity onPress={() => { this.navigateToViewPicture(1) }}>
                
                <Image source={{ uri: 'http://res.cloudinary.com/dy6bbey4u/image/upload/' + this.state.images[1]}}
                  style={styles.picture}/>
              </TouchableOpacity>
              :
              <TouchableOpacity onPress={() => { this.openPickImageModal(1) }} style={styles.placeholderPicture}>
                <ImagePlaceholder style={{ width: pictureWidth, height: pictureWidth }} pictureWidth={pictureWidth} svgPlusNegativeWidth={pictureWidth/5}/>
              </TouchableOpacity>
            }

          </View>

          <View>
            {this.state.mainUserProfile && this.state.images.length == 3 ?
              <TouchableOpacity onPress={() => { this.navigateToViewPicture(2) }}>

                <Image source={{ uri: 'http://res.cloudinary.com/dy6bbey4u/image/upload/' + this.state.images[2]}}
                  style={styles.picture}/>
              </TouchableOpacity>
              :
              <TouchableOpacity onPress={() => { this.openPickImageModal(2) }} style={styles.placeholderPicture}>
                <ImagePlaceholder style={{ width: pictureWidth, height: pictureWidth }} pictureWidth={pictureWidth} svgPlusNegativeWidth={pictureWidth/5}/>
              </TouchableOpacity>

            }

          </View>

        </View>
```

```
import React from 'react';
import { Svg } from 'expo';
const { Circle, Rect, Path, Line, TSpan, TextPat, G } = Svg;



export default function ImagePlaceholder(props) {
  return (
    <Svg style={props.style} fill='rgb(238,238,238)' viewBox={'0 0 ' + props.pictureWidth + ' ' + props.pictureWidth}>
      <Svg.Rect
        x="2" y="2" width={props.pictureWidth - 4} height={props.pictureWidth - 4}
        rx="12" ry="12"
        stroke='#a649f2'
        strokeWidth={3}
        opacity='0.7'
        strokeLinecap="round"
        strokeDasharray={"0, " + props.pictureWidth/8}
      />
      <Svg.Line
        x1={props.pictureWidth/2} y1={props.pictureWidth/2 - props.svgPlusNegativeWidth}
        x2={props.pictureWidth/2} y2={props.pictureWidth/2 + props.svgPlusNegativeWidth}
        stroke="#a649f2"
        strokeWidth="2"
      />
      <Svg.Line
        x1={props.pictureWidth/2 - props.svgPlusNegativeWidth} y1={props.pictureWidth/2}
        x2={props.pictureWidth/2 + props.svgPlusNegativeWidth} y2={props.pictureWidth/2}
        stroke="#a649f2"
        strokeWidth="2"
      />
    </Svg>

  );
}
```
Take a look at the screen:

![rnsvgbug](https://user-images.githubusercontent.com/7046283/55984146-798aa780-5ca5-11e9-9330-e00c25a12667.gif)

In the scenario where there are 2 images, the 3rd item renders an SVG. When I click one of the 2 images, _sometimes_ the SVG is pressed instead. I haven't been able to find a pattern for this, it is completely random when this happens.

Why do I post this on react-native-svg issues?

This only happens if the component is an SVG, it doesn't happen when it is an image for example. This is the second time I encounter this issue and both of them it happened to be a container that had components horizontally with `flexDirection: 'row'`.

**Comment by msand** [comment]
@spyshower What version are you using? A bug in iOS 12 was resolved in v9.2.4
https://github.com/react-native-community/react-native-svg/issues/794#issuecomment-462008318
Can you try adding overflow: hidden to either to the TouchableOpacity or the View containing the Svg?

**Comment by spyshower** [comment]
@msand overflow: 'hidden' fixed it!! Thank you **very** much!

I was using Expo (32) by the way.
